test/integration/api/forms/datasets.js

```diff
@@ -0,0 +1,332 @@
+const { testService } = require('../../setup');
+const testData = require('../../../data/xml');
+
+describe('api: /projects/:id/forms (entity-handling)', () => {
```

Comment:
move the tests in this file up to the outer `datasets.js` test file so they are all together.